heron/stmgr/src/cpp/manager/stmgr-clientmgr.cpp

```diff
@@ -114,6 +117,8 @@ StMgrClient* StMgrClientMgr::CreateClient(const sp_string& _other_stmgr_id,
   options.set_max_packet_size(config::HeronInternalsConfigReader::Instance()
                                   ->GetHeronStreammgrNetworkOptionsMaximumPacketMb() *
                               1024 * 1024);
+  options.set_high_watermark(high_watermark_);
```

Comment:
We have to, because the stmgr unit test cases don't want to read from config, they set these values by themselves.